nat-lab/tests/utils/connection_tracker.py

```diff
@@ -113,6 +114,8 @@ async def execute(self) -> None:
         await self._process.execute(stdout_callback=self.on_stdout)
 
     def get_out_of_limits(self) -> Optional[Dict[str, int]]:
+        if platform.system() == "Darwin":
+            return None
```

Comment:
add simillar line to `execute` method, to prevent from even trying to start conntrack